.pylintrc

```diff
@@ -46,7 +46,7 @@ disable=
 
 good-names=
     # PyLint's default good names.
-    i,j,k,ex,Run,_,
+    i,j,k,s,ex,Run,_,
```

Comment:
Instead of doing this I think we'd be better off just picking a sensible variable name like `string`